src/interpreter/naive_interpreter.nit

```diff
@@ -775,6 +797,27 @@ class MutableInstance
 	var attributes: Map[MAttribute, Instance] = new HashMap[MAttribute, Instance]
 end
 
+# An instance with the original receiver and callsite (for function reference)
+class CallrefInstance
+        super Instance
+
+        # The original receiver
+        #
+        # ~~~~
```

Comment:
use `~~~nitish` to prevent nitunit to run (but still have syntax highlighting)
Also, use `#` instead of `<-` for better syntax highlighting.

See https://nitlanguage.org/tools/nitunit.html